integration_tests/support/commands.ts

```diff
@@ -94,6 +94,20 @@ Cypress.Commands.add('updateShortQuestionSetInductionToArriveOnCheckYourAnswers'
   // Additional Training is the next page. This is asked on the long question set, so this will already have answers set
   Page.verifyOnPage(AdditionalTrainingPage) //
     .submitPage()
+  // 'Has the prisoner worked before' is the next page. This is asked on the long question set so will have answers but
```

Comment:
Updates to some custom cypress commands that drive the screens to create inductions in various combinations